buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -5,7 +5,7 @@ object Versions {
     val ktor = "1.6.5"
 
     val ddmlib = "30.1.2"
-    val dexTestParser = "830520963019a6cefada34fc5eb396003c1468d5" // contains patch https://github.com/linkedin/dex-test-parser/pull/46
+    val dexTestParser = "2.3.4"
```

Comment:
I hope the new version doesn't have any surprises :hiddenpain:

Reply:
It actually has not so many commits, our fix is included in [2.2.1](https://github.com/linkedin/dex-test-parser/releases) and now it's 2.3.4, but they have fixed a quite important [bug](https://github.com/linkedin/dex-test-parser/issues/72) which could affect us.